dockers/docker-snmp/Dockerfile.j2

```diff
@@ -45,6 +45,9 @@ RUN pip3 install --no-cache-dir \
         pyyaml                              \
         smbus
 
+# Install python-redis
+RUN pip3 install redis==4.5.4
```

Comment:
This is already in base image docker-config-engine-bullseye*

Reply:
Fixed, code removed.